packages/create-redwood-app/template/web/src/entry.js

```diff
@@ -7,8 +7,10 @@ import App from './index'
  * prerendering. So React attaches event listeners to the existing markup
  * rather than replacing it.
  * https://reactjs.org/docs/react-dom.html#hydrate
- */
-if (document.getElementById('redwood-app').hasChildNodes()) {
+*/
+const rootElement = document.getElementById('redwood-app')
```

Comment:
Yep, I was wondering about that!

Reply:
Thanks for the tip @thedavidprice. They don't call you el David for no reason!

Reply:
lulz

I kinda want that on a hat